src/gui/src/SoundLibrary/SoundLibraryImportDialog.cpp

```diff
@@ -360,7 +433,40 @@ bool SoundLibraryImportDialog::isSoundLibraryItemAlreadyInstalled( SoundLibraryI
 	return false;
 }
 
+void SoundLibraryImportDialog::loadImage(QString img )
+{
+	QPixmap pixmap;
+	// TODO: Need to make sure that this isn't getting called after user clicks a different item!
```

Comment:
This was a "Done" TODO - removed the comments and committed.
BTW: This is something to watch for in my code... I have a bit of a bad habit of fixing things that I left a TODO comment on and forgetting to remove the TODO